examples/bash_automatic_cd.sh

```diff
@@ -7,14 +7,14 @@
 # To undo the effect of this function, you can type "cd -" to return to the
 # original directory.
 
-function ranger-cd {
-    tempfile="$(mktemp -t tmp.XXXXXX)"
-    ranger --choosedir="$tempfile" "${@:-$(pwd)}"
-    test -f "$tempfile" &&
-    if [ "$(cat -- "$tempfile")" != "$(echo -n `pwd`)" ]; then
-        cd -- "$(cat "$tempfile")"
+ranger-cd() {
+    local temp_file chosen_dir
```

Comment:
Is this basically the only thing we're giving up POSIX compliance for?

Reply:
The only thing in this script that is not POSIX-compliant is use of the `$(<"$filename")` construct which is not available in POSIX sh. I can revert back to `cat` then.

Reply:
`local` is definitely not POSIX and neither is `[[ ]]`.

Reply:
Functions with a dash in their name are neither. Do you want full POSIX compliance?

Reply:
Preferably yes. But it's a lot more important in `ranger.py` of course.